src/autoscaler/api/policyvalidator/policy_validator.go

```diff
@@ -146,7 +148,9 @@ func (pv *PolicyValidator) ParseAndValidatePolicy(rawJson json.RawMessage) (*mod
 	return policy, nil
 }
 
-func (pv *PolicyValidator) validateAttributes(policy *models.ScalingPolicy, result *gojsonschema.Result) {
+// 🚧 To-do: When making the type `models.PolicyDefinition` safe, then this validation should go
```

Comment:
This can be done either in https://github.com/cloudfoundry/app-autoscaler-release/pull/3652 or a new subsequent PR.